src/pcre2_compile.h

```diff
@@ -160,6 +161,10 @@ versions. */
 #define META_FIRST_QUANTIFIER META_ASTERISK
 #define META_LAST_QUANTIFIER  META_MINMAX_QUERY
 
+#define META_ECLASS_OR        0x80440000u  /* || in a class */
+#define META_ECLASS_AND       0x80450000u  /* && in a class */
+#define META_ECLASS_SUB       0x80460000u  /* -- in a class */
```

Comment:
this should go after the last META (ex: META_MINMAX_QUERY), so move up before the convenience aliases (META_FIRST_QUANTIFIER) and special cases.